### PR TITLE
Add 1.6 compliant k8s-backend manifests

### LINKED ISSUE #654: Create KDD + Kubeadm-1.6 compliant YAMLS (add RBAC)
http://docs.projectcalico.org/v2.1/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml

is referenced from here: http://docs.projectcalico.org/v2.1/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml 

But this manifest won't work with k8s 1.6+ because it lacks the RBAC stuff.

### PR DESCRIPTION
**Pull request by djosborne**
Fixes #654 (including temp workaround for https://github.com/projectcalico/calicoctl/issues/1596 )

- Tolerations moved from annotations to toleration field
- Added serviceaccounts
- RBAC roles via additional yaml file
- add workaround for WAITFORDATASTOREISSUE ( https://github.com/projectcalico/calicoctl/issues/1596 )

RBAC file is separate from core yaml as this is not kubeadm specific YAMLS, but general k8s-datstore yamls which should work on any 1.6+ cluster (kubeadm or not).

### CODE REVIEW

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md [hunk 1]
```diff
@@ -25,14 +25,37 @@ You must have a cluster which meets the following requirements:
 
 ## Installation
 
-To install Calico, ensure you have a cluster which meets the above requirements and run the following command:
+To install Calico, ensure you have a cluster which meets the above requirements and run the following command.
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: should probably be "run one of the following commands based on your Kubernetes version"

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md [hunk 2]
```diff
@@ -25,14 +25,37 @@ You must have a cluster which meets the following requirements:
...
 ```
 
 >[Click here to view the above yaml directly.](calico.yaml)
 
+### RBAC
+
+If your Kubernetes cluster has RBAC enabled, you'll need to create RBAC roles for Calico.
+Apply the following manifest to create these RBAC roles.
+
+>Note: Currently, only the above 1.6 manifest assigns service accounts to Calico components.
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: This note isn't technically correct.

I might say this instead.

> Note: The following RBAC policy is compatible with the Kubernetes v1.6 manifest only.

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/1.6/calico.yaml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+              value: "true"
+            # Set Felix endpoint to host default action to ACCEPT.
+            - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
+              value: "ACCEPT"
+            # Disable IPV6 on Kubernetes.
+            - name: FELIX_IPV6SUPPORT
+              value: "false"
+            # Wait for the datastore.
+            - name: WAIT_FOR_DATASTORE
+              value: "false"
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/1.6/calico.yaml]: Once https://github.com/projectcalico/calicoctl/pull/1600/files is merged, let's put this back to "true"

**Comment by djosborne** [comment]
These changes have been rolled into #663

**Pull request closed without merging**